airflow/operators/sql_to_gcs.py

```diff
@@ -252,10 +252,36 @@ def _write_local_schema_file(self, cursor):
             contains the BigQuery schema fields in .json format.
         """
         schema = [self.field_to_bigquery(field) for field in cursor.description]
+        tmp_schema_file_handle = NamedTemporaryFile(delete=True)
```

Comment:
After refactoring, some logic has been moved into `mysql_to_gcs.py`. Suggest update the code as:
```
tmp_schema_file_handle = NamedTemporaryFile(delete=True)
if self.schema is not None and isinstance(self.schema, string_types):
    tmp_schema_file_handle.write(self.schema)
else if self.schema is not None and isinstance(self.schema, list):
    tmp_schema_file_handle.write(json.dumps(self.schema, sort_keys=True).encode('utf-8'))
else:
    schema = [self.field_to_bigquery(field) for field in cursor.description]
    tmp_schema_file_handle.write(json.dumps(schema, sort_keys=True).encode('utf-8'))
```